assets/js/translator.js

```diff
@@ -647,20 +682,7 @@ function populateTranslationList() {
         }
 
         srcLangs = srcLangs.sort(compareLangCodes);
-        dstLangs = dstLangs.sort(function (a, b) {
-            var aPossible = pairs[curSrcLang] && pairs[curSrcLang].indexOf(a) !== -1;
-            var bPossible = pairs[curSrcLang] && pairs[curSrcLang].indexOf(b) !== -1;
-
-            if(aPossible === bPossible) {
-                return compareLangCodes(a, b);
-            }
-            else if(aPossible && !bPossible) {
-                return -1;
-            }
-            else {
-                return 1;
-            }
-        });
+        dstLangs = dstLangs.sort(compareLangCodes);
```

Comment:
Why was this deleted? The extra sorting puts available languages first. It might require some modification for variants properly displaying though.